packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -11,16 +11,16 @@ import {
   useWindowDimensions,
```

Comment:
Can't comment on the appropriate line, but should be able to remove the `ts-nocheck` from the top of the file now, correct?

Reply:
Removed